app/services/sign_in/token_serializer.rb

```diff
@@ -119,5 +119,13 @@ def anti_csrf_token
     def client_config
       @client_config ||= session_container.client_config
     end
+
+    def refresh_token_path
+      @refresh_token_path = if client_config.refresh_token_path.nil?
+                              Constants::Auth::REFRESH_ROUTE_PATH
+                            else
+                              client_config.refresh_token_path
+                            end
```

Comment:
since this is only used once you don't need to set an instance variable. You can keep it how you had before but with the addition of `path` in the method name. 
```ruby
def refresh_token_path
  client_config.refresh_token_path || Constants::Auth::REFRESH_ROUTE_PATH
end
```